apps/condo/domains/ticket/schema/TicketComment.js

```diff
@@ -77,7 +78,7 @@ const TicketComment = new GQLListSchema('TicketComment', {
     plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
     hooks: {
         afterChange: async (requestData) => {
-            await sendDifferentKindsOfNotifications(requestData)
+            await handleTicketCommentEvents(requestData)
```

Comment:
?? `sendDifferentKindsOfNotifications` is more clear naming

Reply:
What about this?

Reply:
fixed